FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -8,22 +8,25 @@
 import SwiftUI
 
 struct HomeView: View {
+
+
+
     // MARK: - Properties
     @EnvironmentObject var quizManager: QuizManager
     @EnvironmentObject var navigationManager: NavigationManager
     @Binding var presentSideMenu: Bool
-    private let levelItem: [LevelItem] = [
+    private let levelItem: [LevelItem] =     [
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/f0556793b45bd971a01f1d6b9fd707367304df54/FlashEnglish/Sources/Views/Home/HomeView.swift#L18"></span>Operators should be surrounded by a single whitespace when they are being used<br />
<code>operator_usage_whitespace</code> <code>HomeView.swift:18</code></td>
    </tr>
  </tbody>
</table>